src/components/LoadingScreen/Splash.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import "./Splash.css";
+export default function Splash() {
+    return (
+        <>
+          <div className="loader-2 center"><span/></div>
```

Comment:
we don't do it like this!!
create a global level Loading component and use redux to active/de-active loading state!!
when redux loading flag is true, loading will appear and when false it will disappear.